styleguide/source/assets/icons/svg/icon-homepage-alert.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should probably use gulp.task('minifyIcons') here to strip extraneous stuff from the svg

Reply:
Running `gulp minifyIcons` has no change.